composer.json

```diff
@@ -13,19 +13,15 @@
       "dev-master": "2.1.x-dev"
     }
   },
-  "repositories": [
-    {
-      "type": "git",
-      "url": "https://github.com/dominicsayers/isemail"
```

Comment:
Why are you removing this? it is needed for the tests comparision :)

Reply:
as mentioned in the commit message, the package is listed on packagist & directly listing the repo causes install issues on windows & git bash.